db/seeds.rb

```diff
@@ -15,18 +15,21 @@
   locale: 1
 )
 
-Book.create(
-  genre: 'Horror',
-  author: 'some author',
-  image: 'some image',
-  title: 'some title',
-  editor: 'some editor',
-  year: '2019'
-)
-
 Rent.create(
   user: User.last, 
   book: Book.last, 
   init_date: Time.now, 
   return_date: Time.now
 )
+
+
+50000.times do
+  Book.create(
```

Comment:
Can we call FactoryBot for this? I know it's not the best approach but we are repeating code if we use Faker like this. Also we are more likely to change the factory of the model than the seed